src/parse-options.js

```diff
@@ -13,6 +17,13 @@ const DEFAULT_OPTIONS = {
 function parseOptions (providedOptions) {
   let options = Object.assign({}, DEFAULT_OPTIONS, providedOptions)
 
+  // normalizing options.color to be a boolean or a color value
+  options.color = (options.color === 'false' || options.color === false) ? false : options.color
+
+  if (!(options.color === false || options.color in styles)) {
+    throw new Error('Invalid color option. Set color to true or to one of the value from https://github.com/chalk/ansi-styles#colors')
```

Comment:
Now that I think about it; we could just inspect `styles` and print them all out right there. That way we don't have to depend on a link.